modules/builds-configuration-file.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * builds/build-configuration.adoc
+
+[id="builds-configuration-file_{context}"]
+= Configuring build settings
+
+You can configure build settings by editing the
+`build.config.openshift.io/cluster` resource.
+
+.Procedure
+
+* Edit the `build.config.openshift.io/cluster` resource:
++
+----
+$ oc edit build.config.openshift.io/cluster
+----
++
+The following is an example `build.config.openshift.io/cluster` resource:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Build<1>
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  creationTimestamp: "2019-05-17T13:44:26Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "107233"
+  selfLink: /apis/config.openshift.io/v1/builds/cluster
+  uid: e2e9cc14-78a9-11e9-b92b-06d6c7da38dc
+spec:
+  additionalTrustedCA:<2>
+    name: myconfigmap
+  buildDefaults:<3>
+    defaultProxy:<4>
+      httpProxy: http://proxy.com
+      httpsProxy: https://proxy.com
+      noProxy: internal.com
+    env:<5>
+    - name: envkey
+      value: envvalue
+    gitProxy:<6>
+      httpProxy: http://gitproxy.com
+      httpsProxy: https://gitproxy.com
+      noProxy: internalgit.com
+    imageLabels:<7>
+    - name: labelkey
+      value: labelvalue
+    resources:<8>
+      limits:
+        cpu: 100m
+        memory: 50Mi
+      requests:
+        cpu: 10m
+        memory: 10Mi
+  buildOverrides:<9>
+    imageLabels:<10>
+    - name: labelkey
+      value: labelvalue
+    nodeSelector:<11>
+      selectorkey: selectorvalue
+    tolerations:<12>
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/builds
+operator: Exists
+----
+<1> `Build`: Holds cluster-wide information on how to handle builds. The
+canonical name is `cluster`.
+<2> `additionalTrustedCA`: A reference to a ConfigMap containing additional CAs that
+should be trusted for image pushes and pulls during builds. The namespace for
+this ConfigMap is `openshift-config`.
+<3> `buildDefaults`: Controls the default information for builds.
+<4> `defaultProxy`: Contains the default proxy settings for all build operations,
+including image pull or push and source download.
+<5> `env`: A set of default environment variables that are applied to the build if
+the specified variables do not exist on the build.
+<6> `gitProxy`: Contains the proxy settings for Git operations only. If set, this
+overrides any Proxy settings for all Git commands, such as `git clone`.
+<7> `imageLabels`: A list of docker labels that are applied to the resulting image.
```

Comment:
Should these instances of "docker" be replaced with "container"?

Reply:
i'd go with `image` not `container` but sure.  (even though "a list of image labels that are applied to the resulting image" feels redundant, it's more accurate than calling it a container)

Reply:
@bparees should we replace docker labels and docker images with container labels and container images? And if so, should the API docs be updated reflect the change? Thanks!

Reply:
`docker label` -> `image label`
`docker image` -> `image` or `container image`

and yes, api docs should be updated too.  this will all make @rhatdan very happy.

but for this particular sentence i'd just change it to "a list of labels that are applied to the resulting image" to avoid the redundancy.